controllers/search-users.js

```diff
@@ -0,0 +1,17 @@
+const User = require('../models/user');
+
+const searchUser = async(req, res) => {
+  try{
+    //Looks for the email in the Users collection
+    const user = await User.find({email: req.body.email});
+    //returns an error message if the user is not found
+    if(!user) return res.send('User not found');
```

Comment:
The User.find() method returns an array. This will equate to true and by negation, an error is never thrown. I suggest you use  !user.length

Reply:
The User.find() returns an object attached with corresponding email, it doesn't return an array. I think it's correct that way.